apps/src/redux/sectionDataRedux.js

```diff
@@ -45,6 +46,7 @@ export const setSection = section => {
     id: section.id,
     script: section.script,
     students: sortedStudents,
+    codeReviewEnabled: section.code_review_enabled,
```

Comment:
For my understanding, can you explain how the section's `code_review_enabled` status gets used (if at all)? It seems like we mostly rely on the @code_review_enabled variable set in script_levels_controller in deciding what to show, is that right?

Reply:
The `@code_review_enabled` variable that gets sent down to the apps code is how we set this variable in redux (in show.js below). `@code_review_enabled` is decided based on the section(s) the student is in. Not sure if that answered your question?